spec/features/post/index.html.erb_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
```

Comment:
- [ ] You need to have a test here that redirects to the post's show page when you click on a post.

Reply:
- [ ] @team as previously requested kindly ensure to add a test that shows redirection to the show page when the user clicks on a post.